### ISSUE TITLE
Request: Support for passing remark configuration

### ISSUE THREAD

**Issue by AxoInsanit** [kt3k/remarker#2]
Hi, it would be nice to have a way to write the remark configuration right in the YML, is this something being considered?
Thanks

**Comment by kt3k** [comment]
Do you mean something like this?

In remarker.yml:

```yml
remarkConfig:
  highlightStyle: molokai
  highlightLines: true
  slideNumberFormat: %current% / %total%
```

The above becomes:

```js
remark.create({
  highlightStyle: 'molokai'
  highlightLines: true
  slideNumberFormat: '%current% / %total%'
})
```

in slides html.

**Comment by kt3k** [comment]
Added the above feature and published as v1.3.0. See [the example](https://github.com/kt3k/remarker/tree/master/examples/remark) for how to right the configuration.

Thanks for the suggestion!

**Comment by AxoInsanit** [comment]
Thanks for implementing it that quickly! 👍
